ni_measurementlink_service/_internal/discovery_client.py

```diff
@@ -27,6 +27,9 @@
     import winerror
 
 _logger = logging.getLogger(__name__)
+# store sub-process that is running; it's only to avoid
+# "ResourceWarning: subprocess N is still running" from subprocess library
+disc_service_subprocess = None
```

Comment:
Naming:
- Don't abbreviate "discovery"
- Prefix with `_` to indicate that this variable is private

Reply:
Updated